e2e/utils.ts

```diff
@@ -41,3 +54,17 @@ export async function startCall(userState: string) {
     return userPage;
 }
 
+export async function startDMWith(userState: string, targetUserName: string) {
+    const userPage = await newUserPage(userState);
+    await userPage.gotoDM(targetUserName);
+    return userPage;
+}
+
+export async function startGMWith(userState: string, targetUserIDs: string[]) {
+    const browser = await chromium.launch();
+    const context = await browser.newContext({storageState: userState});
+    const userPage = new PlaywrightDevPage(await context.newPage());
```

Comment:
Any reason for not using `newUserPage()` here?

Reply:
I guess I didn't remember? :)